staging/src/k8s.io/api/apiserverinternal/v1alpha1/doc.go

```diff
@@ -16,12 +16,8 @@ limitations under the License.
 
 // +k8s:deepcopy-gen=package
 // +k8s:protobuf-gen=package
-// +k8s:conversion-gen=k8s.io/apiserver/pkg/apis/apiserverinternal
 // +k8s:openapi-gen=true
-// +k8s:defaulter-gen=TypeMeta
-
-// +groupName=internal.apiserver.k8s.io
```

Comment:
I expect this line is kept, like https://github.com/kubernetes/kubernetes/blob/adefb2ebc8866bac3b17a6e7bbdf33a31128ec3d/staging/src/k8s.io/api/flowcontrol/v1alpha1/doc.go#L21